qutip/qobj.py

```diff
@@ -1577,21 +1596,29 @@ def matrix_element(self, bra, ket):
 
         Parameters
         -----------
-        bra : :class:`qutip.Qobj`
-            Quantum object of type 'bra' or 'ket'
+        bra : :obj:`.Qobj`
+            Quantum object of type :obj:`.bra`
 
-        ket : :class:`qutip.Qobj`
-            Quantum object of type 'ket'.
+        ket : :obj:`.Qobj`
+            Quantum object of type :obj:`.ket`.
 
         Returns
         -------
-        elem : complex
+        complex
             Complex valued matrix element.
 
+        Raises
+        -------
+        TypeError : Can only get matrix elements for an operator
+            When input is not a valid operator.
+
+        TypeError : Can only calculate matrix elements for bra  and ket vectors
+            When input is not a valid ket or bra vector.
+
         Notes
         -----
-        It is slightly more computationally efficient to use a ket
-        vector for the 'bra' input.
+        It is slightly more computationally efficient to use a `ket`
+        vector for the `bra` input.
```

Comment:
the "ket" should be left with no formatting (it's just a word), and `bra` should have 2 backticks to make it monospaced rather than italic